setup.py

```diff
@@ -56,8 +56,14 @@ def run(self):
         build_py.run(self)
 
 
+def get_git_tag():
+    # probably not valid subprocess usage, but you get the idea
+    import subprocess
+    return subprocess.run('git tag -l | HEAD -1').stdout
+
+
 setup(name='citrine',
-      version='0.16.4',
+      version=get_git_tag(),
```

Comment:
Does this work with pypi-hosted releases?  Namely, if this is released to and then installed from pypi, do I get an error?  Does `pip show` work?

Reply:
While we don't currently, I might expect that we want to expose the client version as something like:
```python
client = Citrine(...)
print(client.version)
```
Does this pattern play nice with that?